src/System.Collections.Specialized/tests/StringDictionary/GetItemStrTests.cs

```diff
@@ -3,216 +3,35 @@
-            if (sd.Count != len)
-            {
-                Assert.False(true, string.Format("Error, count is {0} instead of {1}", sd.Count, len));
-            }
-
-            for (int i = 0; i < len; i++)
-            {
-                if (String.Compare(sd[keys[i]], values[i]) != 0)
-                {
-                    Assert.False(true, string.Format("Error, returned {1} instead of {2}", i, sd[keys[i]], values[i]));
-                }
-            }
-
-
-
-            //
-            //  [] get Item on dictionary with identical values
-            //
-
-            sd.Clear();
-            string intlStr = intl.GetRandomString(MAX_LEN);
-
-            sd.Add("keykey1", intlStr);        // 1st duplicate
-            for (int i = 0; i < len; i++)
-            {
-                sd.Add(keys[i], values[i]);
-            }
-            sd.Add("keykey2", intlStr);        // 2nd duplicate
-            if (sd.Count != len + 2)
-            {
-                Assert.False(true, string.Format("Error, count is {0} instead of {1}", sd.Count, len + 2));
-            }
-
-            // get Item
-            //
-            if (String.Compare(sd["keykey1"], intlStr) != 0)
-            {
-                Assert.False(true, string.Format("Error, returned {1} instead of {2}", sd["keykey1"], intlStr));
-            }
-
-            if (String.Compare(sd["keykey2"], intlStr) != 0)
-            {
-                Assert.False(true, string.Format("Error, returned {1} instead of {2}", sd["keykey2"], intlStr));
-            }
-
-            //
-            // Intl strings
-            // [] get Item() on dictionary filled with Intl strings
-            //
-
-            string[] intlValues = new string[len * 2];
-            // fill array with unique strings
-            //
-            for (int i = 0; i < len * 2; i++)
-            {
-                string val = intl.GetRandomString(MAX_LEN);
-                while (Array.IndexOf(intlValues, val) != -1)
-                    val = intl.GetRandomString(MAX_LEN);
-                intlValues[i] = val;
-            }
-
-            //
-            // will use first half of array as values and second half as keys
-            //
-            sd.Clear();
-            for (int i = 0; i < len; i++)
-            {
-                sd.Add(intlValues[i + len], intlValues[i]);
-            }
-            if (sd.Count != len)
-            {
-                Assert.False(true, string.Format("Error, count is {0} instead of {1}", sd.Count, len));
-            }
-
-            // get item
-            for (int i = 0; i < len; i++)
-            {
-                if (String.Compare(sd[intlValues[i + len]], intlValues[i]) != 0)
-                {
-                    Assert.False(true, string.Format("Error, returned {1} instead of {2}", i, sd[intlValues[i + len]], intlValues[i]));
-                }
-            }
-
-            //
-            // [] Case sensitivity: keys are always lowercased
-            //
```

Comment:
That's the case for all of the test files it would seem. The old tests have some scenarios (Intl strings, empty stringdictionary, full stringdictionary, lowercase keys, etc) that are tested for every  method but in the new tests we're missing a few of those old scenarios. 

I think a case sensitivity test at the very least would be good to have for each method.